kcqrs-testing/src/main/kotlin/com/clouway/kcqrs/testing/InMemoryAggregateRepository.kt

```diff
@@ -37,10 +37,13 @@ class InMemoryAggregateRepository : AggregateRepository {
     override fun <T : AggregateRoot> getByIds(ids: List<String>, type: Class<T>): Map<String, T> {
         val foundAggregates = mutableMapOf<String, T>()
         ids.forEach {
```

Comment:
return ids.filter { aggregateIdToEvents.containsKey(it) }.map { .... }.toMap

Reply:
changed